onnxruntime/core/optimizer/transpose_optimizer/transpose_optimizer.cc

```diff
@@ -942,36 +942,6 @@ void PermuteInput(api::GraphRef& graph, api::NodeRef& node, size_t i, const std:
   node.SetInput(i, gather_output);
 }
 
-static bool HandleResize(HandlerArgs& args) {
-  auto inputs = args.node.Inputs();
```

Comment:
The transpose optimizer is intended for use with multiple EPs, so a limitation of the ORT CPU EP implementation ideally shouldn't be handled by removing all support for transposing Resize.

Can the ORT upsample logic implicitly determine if it's NHWC by checking the scales and the dim values? not sure if there's any scenario where the value of C would be larger than H or W and the scale for the C dim would assumably always be 1.

If that's not possible and there's no simple way to special case Resize for now, it would be better to keep the code and unit tests and comment them out so that the functionality can be easily re-instated when we have a better solution.

Reply:
The CPU implementation should be refactored to allow for blindly addressing either NHWC or NCHW, but now let's just comment the handler for release.

Reply:
This is holding up the release. @skottmckay if this change is not blocking any customer I'm inclined to go with the commenting option.

Reply:
Commented.